packages/terafoundation_kafka_connector/src/schema.ts

```diff
@@ -38,5 +38,10 @@ export default {
         doc: 'Private key passphrase',
         default: undefined,
         format: 'optional_String'
+    },
+    rdkafka_options: {
+        doc: 'Additional rdkafka configuration options. See https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md for available options.',
+        default: undefined,
+        format: Object
```

Comment:
Do we need to do a more complex check here? What happens if we pass in some random object?